### ISSUE TITLE
import "tflite" model into pytorch 

### ISSUE THREAD

**Issue by peter197321** [pytorch/pytorch#39232]
Hello,

I am newbie to PyTorch and therefore question on use: I'd like to import "tflite" model either float32 or quantized, how or what approach to take to import in such a pre-trained model? 
Can you please point me to some examples?

Thank you.

**Comment by 1e100** [comment]
This isn't really doable in the general case. TFLite is an inference format, and as such it does some things during conversion that are impossible to undo and make the model trainable again (constant folding, batchnorm fusion, etc).

If you are not interested in being able to train a model, then you could conceivably read the weights from *.tflite file (which is a FlatBuffer that uses this schema https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/schema/schema.fbs and import those weights into a hand-coded model on the PyTorch side. If you do this, you will also likely have to emulate the "SAME" padding mode before your model is able to produce the same result.

TL;DR: _technically_ it can be done, but it's not automatic, it's a lot of work, and the result will not, generally, be trainable. HTH.

**Comment by gchanan** [comment]
Please use the forums (https://discuss.pytorch.org/) for questions -- we try to limit github to bugs / feature requests.
